ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -217,7 +217,7 @@ releases:
   latest:
     integration:
       include_built_images: true
-      name: "4.10"
```

Comment:
it might mean for disconnected job to start failing on master
would we want to handle it? (e.g., keep it using 4.10 for now)

Reply:
Why? Is disconnected installation not supported in 4.12 anymore? Or is there a bug that we should be aware of?

Reply:
because of a bug we have in 4.11 (and I think should exist in 4.12)
I think the bug is already filed, but not handled iirc

Reply:
Maybe we can specify that disconnected job will run in 4.10 instead of latest?

Reply:
sounds good to me (as a temporal workaround)

Reply:
Do you remember if both `e2e-ai-operator-ztp-disconnected-periodic` and `e2e-ai-operator-ztp-disconnected` need to be changed to 4.10?

Reply:
yes both of them need a <4.11 based version